.jenkins/dsl/jobs.groovy

```diff
@@ -69,48 +98,67 @@ if (!isMainBranch()) {
 def otherFolder = KogitoConstants.KOGITO_DSL_OTHER_FOLDER
 if (isMainBranch()) {
     folder(otherFolder)
-    setupOptaPlannerTurtleTestsJob(otherFolder);
+    setupOptaPlannerTurtleTestsJob(otherFolder)
 }
 
 /////////////////////////////////////////////////////////////////
 // Methods
 /////////////////////////////////////////////////////////////////
 
-void setupOptaplannerPrJob(String jobFolder) {
+void setupOptaplannerPrJob() {
     def jobParams = getDefaultJobParams()
-    jobParams.job.folder = jobFolder
-    jobParams.pr = [ blackListTargetBranches: ['7.x'] ]
+    jobParams.pr.whiteListTargetBranches = ['8.5.x', '8.7.x']
```

Comment:
Does this `whiteListTargetBranches` (and other below) require manual updating for every new (product) release?

Reply:
this is introduced just temporarely as branches 8.5.x and 8.7.x cannot support the multijob pipeline due to Jenkinsfile not updated there. Once those branches are not supported anymore, then we can remove this line and also the "old" PR check mentioned here: https://github.com/kiegroup/optaplanner/pull/1304/files#diff-954fb9904df55eadd8693b9166aadf6ebb64b995f8c366ed1a1b7be597aeacb3R63-R66

Reply:
In that case, would you mind adding a TODO note there making it clear what and when should be removed?

Reply:
added